README.md

```diff
@@ -264,7 +269,6 @@ These builtins involve floating-point types ("`f128`", "`f80`" and complex numbe
 - ~~ppc/gcc_qmul.c~~
 - ~~ppc/gcc_qsub.c~~
 - ~~ppc/multc3.c~~
-- ~~subtf3.c~~
 - ~~trunctfdf2.c~~
 - ~~trunctfsf2.c~~
```

Comment:
Are you sure these aren't needed? I would expect all the `tf` builtins to be needed for `f128`.

Reply:
They are, I just wanted to get arithmetic passing before working on those.